api/src/main/java/org/apache/cloudstack/api/command/user/template/ListTemplatesCmd.java

```diff
@@ -78,6 +78,9 @@
     @Parameter(name = ApiConstants.PARENT_TEMPLATE_ID, type = CommandType.UUID, entityType = TemplateResponse.class, description = "list datadisk templates by parent template id", since = "4.4")
     private Long parentTemplateId;
 
+    @Parameter(name = ApiConstants.SHOW_UNIQUE, type = CommandType.BOOLEAN, description = "If set to true, list only unique templates across zones")
```

Comment:
@davidjumani add version `4.15`

Reply:
@rhtyd this PR is aiming 4.13 branch. Considering the version 4.15, shouldn`t it aim master?